diff --git a/librewolf.cfg b/librewolf.cfg
index 1ad0cea..8fe76be 100644
--- a/librewolf.cfg
+++ b/librewolf.cfg
@@ -389,7 +389,7 @@ defaultPref("extensions.ui.experiment.hidden", false);
 lockPref("dom.indexedDB.enabled", true); //default true
 //lockPref("dom.indexedDB.logging.details", false); //default true
 //lockPref("dom.indexedDB.logging.enabled", false); //default true
-lockPref("dom.w3c_pointer_events.enabled", false);
+lockPref("dom.w3c_pointer_events.enabled", true);
 //lockPref("network.http.spdy.enabled", false);
 //lockPref("network.http.spdy.enabled.deps", false);
 //lockPref("network.http.spdy.enabled.http2", false);
@@ -1132,7 +1132,7 @@ lockPref("prio.publicKeyB", "");
 // >>>>>>>>>>>>>>>>>>>>
 
 // Pref : Other webGl [WINDOWS]
-lockPref("webgl.dxgl.enabled", false);
+//lockkPref("webgl.dxgl.enabled", false);
 
 // Pref : disable scanning for plugins [WINDOWS]
 lockPref("plugin.scan.plid.all", false);
@@ -1365,7 +1365,7 @@ defaultPref("layout.frame_rate.precise", true);
 // It's likely your browser is already set to use these features.
 // May introduce instability on some hardware.
 // Tor compatibility - have inverted values in tor.
-defaultPref("webgl.force-enabled", true);
+//defaultPref("webgl.force-enabled", true);
 defaultPref("layers.acceleration.force-enabled", true);
 
 // Pref : 2508: disable hardware acceleration to reduce graphics fingerprinting
@@ -1432,16 +1432,16 @@ defaultPref("browser.tabs.animate", false);
 // WebGL introduces high fingerprinting (WebGL is a js API for directly accessing hardware)
 defaultPref("webgl.disabled", false); // Tor have it false but the rest is the same (webgl)
 // This does not leak
-lockPref("webgl.enable-webgl2", false);
-lockPref("webgl.min_capability_mode", true);
+//lockPref("webgl.enable-webgl2", false);
+//lockPref("webgl.min_capability_mode", true);
 
 // Bench Diff : 0/5000
 // Pref : Disable webGL II/II
 // WebGL introduces high fingerprinting (WebGL is a js API for directly accessing hardware)
-lockPref("pdfjs.enableWebGL", false);
-lockPref("webgl.disable-extensions", true);
-lockPref("webgl.disable-fail-if-major-performance-caveat", true);
-lockPref("webgl.enable-debug-renderer-info", false); //Deprecated Active
+//lockPref("pdfjs.enableWebGL", false);
+//lockPref("webgl.disable-extensions", true);
+//lockPref("webgl.disable-fail-if-major-performance-caveat", true);
+//lockPref("webgl.enable-debug-renderer-info", false); //Deprecated Active
 
 // Bench Diff : -1500/5000
 // Pref : Disable asm.js
@@ -1804,7 +1804,7 @@ lockPref("datareporting.healthreport.uploadEnabled", false);
 lockPref("datareporting.policy.dataSubmissionEnabled", false);
 
 // Pref : Disable right-click menu manipulation via JavaScript (disabled)
-defaultPref("dom.event.contextmenu.enabled", false);
+defaultPref("dom.event.contextmenu.enabled", true);
 
 // Pref : Disable clipboard event detection (onCut/onCopy/onPaste) via Javascript
 // Disabling clipboard events breaks Ctrl+C/X/V copy/cut/paste functionaility in
