diff --git a/librewolf.cfg b/librewolf.cfg
index 03524fd..7aa8d33 100644
--- a/librewolf.cfg
+++ b/librewolf.cfg
@@ -375,6 +375,7 @@ defaultPref("network.proxy.socks_version", 5);
 defaultPref("browser.tabs.loadBookmarksInTabs", true);
 defaultPref("devtools.debugger.remote-enabled", false);
 defaultPref("devtools.chrome.enabled", false);
+defaultPref("general.useragent.site_specific_overrides", false);
 defaultPref("extensions.ui.experiment.hidden", false);
 //defaultPref("extensions.ui.dictionary.hidden", false);
 //defaultPref("extensions.ui.locale.hidden", false);
@@ -388,7 +389,7 @@ defaultPref("extensions.ui.experiment.hidden", false);
 lockPref("dom.indexedDB.enabled", true); //default true
 //lockPref("dom.indexedDB.logging.details", false); //default true
 //lockPref("dom.indexedDB.logging.enabled", false); //default true
-lockPref("dom.w3c_pointer_events.enabled", false);
+lockPref("dom.w3c_pointer_events.enabled", true);
 //lockPref("network.http.spdy.enabled", false);
 //lockPref("network.http.spdy.enabled.deps", false);
 //lockPref("network.http.spdy.enabled.http2", false);
@@ -616,7 +617,7 @@ defaultPref("extensions.webextensions.background-delayed-startup", true); //defa
 // >>>>>>>>>>>>>>>>>>>>
 
 // Pref : Disabling performance addon url [FF64+]
-lockPref("devtools.performance.recording.ui-base-url", "http://localhost:4242");
+lockPref("devtools.performance.recording.ui-base-url", "https://localhost:4242");
 // Default Value : https://perf-html.io
 
 // Pref : The default set of protocol handlers for irc [FF64+]
@@ -881,12 +882,12 @@ lockPref("extensions.getAddons.compatOverides.url", "");
 // https://services.addons.mozilla.org/api/v3/addons/compat-override/?guid=%IDS%&lang=%LOCALE%
 
 // Pref :
-defaultPref("extensions.getAddons.get.url", "");
+defaultPref("extensions.getAddons.get.url", "https://services.addons.mozilla.org/api/v3/addons/search/?guid=%IDS%&lang=%LOCALE%");
 // Default Value
 // https://services.addons.mozilla.org/api/v3/addons/search/?guid=%IDS%&lang=%LOCALE%
 
 // Pref :
-defaultPref("extensions.getAddons.langpacks.url", "");
+defaultPref("extensions.getAddons.langpacks.url", "https://services.addons.mozilla.org/api/v3/addons/language-tools/?app=firefox&type=language&appversion=%VERSION%");
 // Default Value
 // https://services.addons.mozilla.org/api/v3/addons/language-tools/?app=firefox&type=language&appversion=%VERSION%
 
@@ -896,12 +897,12 @@ defaultPref("extensions.getAddons.link.url", "");
 // https://addons.mozilla.org/%LOCALE%/firefox/
 
 // Pref :
-defaultPref("extensions.getAddons.search.browseURL", "");
+defaultPref("extensions.getAddons.search.browseURL", "https://addons.mozilla.org/%LOCALE%/firefox/search?q=%TERMS%&platform=%OS%&appver=%VERSION%");
 // Default Value
 // https://addons.mozilla.org/%LOCALE%/firefox/search?q=%TERMS%&platform=%OS%&appver=%VERSION%
 
 // Pref :
-lockPref("extensions.getAddons.themes.browseURL", "");
+lockPref("extensions.getAddons.themes.browseURL", "https://addons.mozilla.org/%LOCALE%/firefox/themes/?src=firefox");
 // Default Value
 // https://addons.mozilla.org/%LOCALE%/firefox/themes/?src=firefox
 
@@ -931,7 +932,7 @@ lockPref("browser.newtabpage.activity-stream.fxaccounts.endpoint", "");
 // https://accounts.firefox.com/
 
 // Pref :
-defaultPref("extensions.update.url", "");
+//defaultPref("extensions.update.url", "");
 // Default Value
 // https://versioncheck.addons.mozilla.org/update/VersionCheck.php?reqVersion=
 // %REQ_VERSION%&id=%ITEM_ID%&version=%ITEM_VERSION%&maxAppVersion=
@@ -1476,7 +1477,7 @@ lockPref("javascript.options.shared_memory", false);
 // So overall, disabling SW should enhance performance because it blocks SW Scripts.
 // [NOTE] Service worker APIs are hidden (in Firefox) and cannot be used when in PB mode.
 // [NOTE] Service workers only run over HTTPS. Service Workers have no DOM access.
-defaultPref("dom.serviceWorkers.enabled", false);
+lockPref("dom.serviceWorkers.enabled", false);
 
 // >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
 // Section : General Settings 1/3
@@ -1803,7 +1804,7 @@ lockPref("datareporting.healthreport.uploadEnabled", false);
 lockPref("datareporting.policy.dataSubmissionEnabled", false);
 
 // Pref : Disable right-click menu manipulation via JavaScript (disabled)
-defaultPref("dom.event.contextmenu.enabled", false);
+defaultPref("dom.event.contextmenu.enabled", true);
 
 // Pref : Disable clipboard event detection (onCut/onCopy/onPaste) via Javascript
 // Disabling clipboard events breaks Ctrl+C/X/V copy/cut/paste functionaility in
@@ -1893,7 +1894,7 @@ lockPref("dom.enable_user_timing", false);
 // Pref : Disable Web Audio API
 // https://bugzilla.mozilla.org/show_bug.cgi?id=1288359
 // Avoid fingerprinting
-defaultPref("dom.webaudio.enabled", false);
+lockPref("dom.webaudio.enabled", false);
 
 // Pref : When geolocation is enabled, don't log geolocation requests to the console
 lockPref("geo.wifi.logging.enabled", false);
@@ -2772,3 +2773,10 @@ defaultPref("devtools.selfxss.count", 0);
 // enable HTTPS only mode by default
 defaultPref("dom.security.https_only_mode", true);
 defaultPref("dom.security.https_only_mode_ever_enabled", true);
+
+
+// attempting new cookie behavior (use Settings > Cookies and Site Data > Manage Exceptions)
+// (please not that after you added a site, you must close LibreWolf before the cookies/etc are actually saved.)
+defaultPref("privacy.clearOnShutdown.cookies", false);
+defaultPref("privacy.clearOnShutdown.offlineApps", false);
+defaultPref("network.cookie.lifetimePolicy", 2);
