diff --git a/librewolf.cfg b/librewolf.cfg
index 1ad0cea..0d9df09 100644
--- a/librewolf.cfg
+++ b/librewolf.cfg
@@ -389,7 +389,7 @@ defaultPref("extensions.ui.experiment.hidden", false);
 lockPref("dom.indexedDB.enabled", true); //default true
 //lockPref("dom.indexedDB.logging.details", false); //default true
 //lockPref("dom.indexedDB.logging.enabled", false); //default true
-lockPref("dom.w3c_pointer_events.enabled", false);
+lockPref("dom.w3c_pointer_events.enabled", true);
 //lockPref("network.http.spdy.enabled", false);
 //lockPref("network.http.spdy.enabled.deps", false);
 //lockPref("network.http.spdy.enabled.http2", false);
@@ -932,7 +932,7 @@ lockPref("browser.newtabpage.activity-stream.fxaccounts.endpoint", "");
 // https://accounts.firefox.com/
 
 // Pref :
-defaultPref("extensions.update.url", "");
+//defaultPref("extensions.update.url", "");
 // Default Value
 // https://versioncheck.addons.mozilla.org/update/VersionCheck.php?reqVersion=
 // %REQ_VERSION%&id=%ITEM_ID%&version=%ITEM_VERSION%&maxAppVersion=
@@ -1804,7 +1804,7 @@ lockPref("datareporting.healthreport.uploadEnabled", false);
 lockPref("datareporting.policy.dataSubmissionEnabled", false);
 
 // Pref : Disable right-click menu manipulation via JavaScript (disabled)
-defaultPref("dom.event.contextmenu.enabled", false);
+defaultPref("dom.event.contextmenu.enabled", true);
 
 // Pref : Disable clipboard event detection (onCut/onCopy/onPaste) via Javascript
 // Disabling clipboard events breaks Ctrl+C/X/V copy/cut/paste functionaility in
@@ -2773,3 +2773,10 @@ defaultPref("devtools.selfxss.count", 0);
 // enable HTTPS only mode by default
 defaultPref("dom.security.https_only_mode", true);
 defaultPref("dom.security.https_only_mode_ever_enabled", true);
+
+
+// attempting new cookie behavior (use Settings > Cookies and Site Data > Manage Exceptions)
+// (please not that after you added a site, you must close LibreWolf before the cookies/etc are actually saved.)
+defaultPref("privacy.clearOnShutdown.cookies", false);
+defaultPref("privacy.clearOnShutdown.offlineApps", false);
+defaultPref("network.cookie.lifetimePolicy", 2);
