diff --git a/librewolf.cfg b/librewolf.cfg
index 1ad0cea..597dd5e 100644
--- a/librewolf.cfg
+++ b/librewolf.cfg
@@ -264,11 +264,12 @@ defaultPref("media.gmp-manager.certs.1.commonName", "");
 // User Settings : WebRTC
 // ----------------------
 
-defaultPref("media.navigator.enabled", false);
-defaultPref("media.navigator.video.enabled", false);
-defaultPref("media.getusermedia.browser.enabled", false);
-defaultPref("media.getusermedia.screensharing.enabled", false);
-defaultPref("media.getusermedia.audiocapture.enabled", false);
+// these 5 are needed to be true for webcam/camera
+defaultPref("media.navigator.enabled", true);
+defaultPref("media.navigator.video.enabled", true);
+defaultPref("media.getusermedia.browser.enabled", true);
+defaultPref("media.getusermedia.screensharing.enabled", true);
+defaultPref("media.getusermedia.audiocapture.enabled", true);
 defaultPref("media.peerconnection.use_document_iceservers", false);
 defaultPref("media.peerconnection.identity.enabled", false);
 // 10000 per default
@@ -389,7 +390,7 @@ defaultPref("extensions.ui.experiment.hidden", false);
 lockPref("dom.indexedDB.enabled", true); //default true
 //lockPref("dom.indexedDB.logging.details", false); //default true
 //lockPref("dom.indexedDB.logging.enabled", false); //default true
-lockPref("dom.w3c_pointer_events.enabled", false);
+lockPref("dom.w3c_pointer_events.enabled", true);
 //lockPref("network.http.spdy.enabled", false);
 //lockPref("network.http.spdy.enabled.deps", false);
 //lockPref("network.http.spdy.enabled.http2", false);
@@ -882,12 +883,12 @@ lockPref("extensions.getAddons.compatOverides.url", "");
 // https://services.addons.mozilla.org/api/v3/addons/compat-override/?guid=%IDS%&lang=%LOCALE%
 
 // Pref :
-defaultPref("extensions.getAddons.get.url", "");
+defaultPref("extensions.getAddons.get.url", "https://services.addons.mozilla.org/api/v3/addons/search/?guid=%IDS%&lang=%LOCALE%");
 // Default Value
 // https://services.addons.mozilla.org/api/v3/addons/search/?guid=%IDS%&lang=%LOCALE%
 
 // Pref :
-defaultPref("extensions.getAddons.langpacks.url", "");
+defaultPref("extensions.getAddons.langpacks.url", "https://services.addons.mozilla.org/api/v3/addons/language-tools/?app=firefox&type=language&appversion=%VERSION%");
 // Default Value
 // https://services.addons.mozilla.org/api/v3/addons/language-tools/?app=firefox&type=language&appversion=%VERSION%
 
@@ -897,12 +898,12 @@ defaultPref("extensions.getAddons.link.url", "");
 // https://addons.mozilla.org/%LOCALE%/firefox/
 
 // Pref :
-defaultPref("extensions.getAddons.search.browseURL", "");
+defaultPref("extensions.getAddons.search.browseURL", "https://addons.mozilla.org/%LOCALE%/firefox/search?q=%TERMS%&platform=%OS%&appver=%VERSION%");
 // Default Value
 // https://addons.mozilla.org/%LOCALE%/firefox/search?q=%TERMS%&platform=%OS%&appver=%VERSION%
 
 // Pref :
-lockPref("extensions.getAddons.themes.browseURL", "");
+lockPref("extensions.getAddons.themes.browseURL", "https://addons.mozilla.org/%LOCALE%/firefox/themes/?src=firefox");
 // Default Value
 // https://addons.mozilla.org/%LOCALE%/firefox/themes/?src=firefox
 
@@ -932,7 +933,7 @@ lockPref("browser.newtabpage.activity-stream.fxaccounts.endpoint", "");
 // https://accounts.firefox.com/
 
 // Pref :
-defaultPref("extensions.update.url", "");
+//defaultPref("extensions.update.url", "");
 // Default Value
 // https://versioncheck.addons.mozilla.org/update/VersionCheck.php?reqVersion=
 // %REQ_VERSION%&id=%ITEM_ID%&version=%ITEM_VERSION%&maxAppVersion=
@@ -1804,13 +1805,13 @@ lockPref("datareporting.healthreport.uploadEnabled", false);
 lockPref("datareporting.policy.dataSubmissionEnabled", false);
 
 // Pref : Disable right-click menu manipulation via JavaScript (disabled)
-defaultPref("dom.event.contextmenu.enabled", false);
+defaultPref("dom.event.contextmenu.enabled", true);
 
 // Pref : Disable clipboard event detection (onCut/onCopy/onPaste) via Javascript
 // Disabling clipboard events breaks Ctrl+C/X/V copy/cut/paste functionaility in
 // JS-based web applications (Google Docs etc.)
 // https://developer.mozilla.org/en-US/docs/Mozilla/Preferences/Preference_reference/dom.event.clipboardevents.enabled
-lockPref("dom.event.clipboardevents.enabled", false);
+lockPref("dom.event.clipboardevents.enabled", true);
 
 // Pref : Force Punycode for Internationalized Domain Names
 // http://kb.mozillazine.org/Network.IDN_show_punycode
@@ -2773,3 +2774,14 @@ defaultPref("devtools.selfxss.count", 0);
 // enable HTTPS only mode by default
 defaultPref("dom.security.https_only_mode", true);
 defaultPref("dom.security.https_only_mode_ever_enabled", true);
+
+
+// attempting new cookie behavior (use Settings > Cookies and Site Data > Manage Exceptions)
+// (please not that after you added a site, you must close LibreWolf before the cookies/etc are actually saved.)
+defaultPref("privacy.clearOnShutdown.cookies", false);
+defaultPref("privacy.clearOnShutdown.offlineApps", false);
+defaultPref("network.cookie.lifetimePolicy", 2);
+
+// attempting to allow webcam and microphone
+defaultPref("permissions.default.microphone",1);
+defaultPref("permissions.default.camera",1);
