diff --git a/python/mozbuild/mozbuild/mach_commands.py b/python/mozbuild/mozbuild/mach_commands.py
index 10d7a8f..0ce2ee3 100644
--- a/python/mozbuild/mozbuild/mach_commands.py
+++ b/python/mozbuild/mozbuild/mach_commands.py
@@ -2457,6 +2457,7 @@ def package_l10n(command_context, verbose=False, locales=[]):
         )
         command_context.run_process(
             [
+                "python3",
                 mozpath.join(command_context.topsrcdir, "mach"),
                 "build",
                 "chrome-{}".format(locale),
